src/components/Navigation/NavigationItem/_navigationItem.scss

```diff
@@ -65,9 +65,39 @@
   flex-direction: column;
   gap: $spacing-sm;
 
-  &--large-cta {
+  &--left,
+  &--right {
+    &__title {
+      margin-bottom: 0 !important;
```

Comment:
it it necessary to use `!important` ?

Reply:
I think it is necessary because the text component automatically applies margin to the h tags being rendered and I can't override it from styles here without it.

Reply:
Fixed 😝